examples/conanfile/package_id/settings_and_options.rst

```diff
@@ -0,0 +1,242 @@
+.. _examples_conanfile_package_id_settings_options:
+
+How settings and options of a recipe influence its package ID
+=============================================================
+
+In Conan, a package ID is a unique identifier for a package binary that takes into account all the factors that affect its contents.
+These factors include recipe options and settings as well as requirements or tool requirements.
+
+Let's see how settings and options affect the package ID and some examples where they should not.
+
+How settings influence the package ID
+-------------------------------------
+
+Settings are development project-wide variables, like the compiler, its version, or the OS itself.
+These variable values have to be defined, they should match the values of our development environment, and they cannot have a default value.
```

Comment:
I did not understand the default value thing in the beginning, maybe this change makes it clearer.
```suggestion
These variable values have to be defined, they should match the values of our development environment, and they cannot have a default value like options do.
```